src/Generator/Generators/CLI/CLIHeadersTemplate.cs

```diff
@@ -137,6 +137,9 @@ public void GenerateForwardRefs()
             var typeReferences = typeReferenceCollector.TypeReferences;
             var @namespace = ConvertForwardReferencesToNamespaces(typeReferences);
 
+            foreach (var cls in @namespace.Classes)
+                cls.IsStruct = ExtensionMethods.CheckClassIsStructible(cls, Driver);
```

Comment:
Should I add a new pass? I can't find any pass that would be fit to accommodate this.
Also, I can add a new file for the pass but do I actually get it working? Please tell me the file where it is defined as to which passes will be applicable and which not for a certain test project.